docs/reference/integrations.md

```diff
@@ -50,3 +50,18 @@ to the dashboard can be made but will not be persisted upon restart/redeployment
 
 Grafana-Prometheus relate command: `juju relate grafana-k8s:grafana-source prometheus-k8s:grafana-source`  
 Grafana-dashboard relate command: `juju relate wordpress-k8s grafana-dashboard`
+
+### database:
+
+_Interface_: mysql_client
+_Supported charms_: [Charmed MySQL](https://github.com/canonical/mysql-operator)
```

Comment:
Shouldn't we link to charmhub instead of github ?
Also we could reference Charmed Mysql K8S

Reply:
I agree with both of Gregory's comments here